core/lib/web3_decl/src/namespaces/unstable.rs

```diff
@@ -38,10 +39,11 @@ pub trait UnstableNamespace {
         tee_type: Option<TeeType>,
     ) -> RpcResult<Vec<TeeProof>>;
 
+    // Accepts batch_or_block_number as a JSON number (for L1 batch) or an object `{"block": <number>}` (for GW block).
     #[method(name = "getChainLogProof")]
     async fn get_chain_log_proof(
```

Comment:
could we split these two methods? batch or block number looks suspicious from ux perspective